src/parser/mage/fire/__snapshots__/CombatLogParser.test.js.snap

```diff
@@ -3391,17 +3391,29 @@ exports[`The Fire Mage Analyzer analyzers Combatants matches the statistic snaps
 
 exports[`The Fire Mage Analyzer analyzers Combatants matches the suggestions snapshot 1`] = `Array []`;
 
-exports[`The Fire Mage Analyzer analyzers CombustionCharges matches the statistic snapshot 1`] = `"module has no statistic method"`;
+exports[`The Fire Mage Analyzer analyzers Combustion matches the statistic snapshot 1`] = `"module has no statistic method"`;
 
-exports[`The Fire Mage Analyzer analyzers CombustionCharges matches the suggestions snapshot 1`] = `Array []`;
-
-exports[`The Fire Mage Analyzer analyzers CombustionFirestarter matches the statistic snapshot 1`] = `"module inactive"`;
-
-exports[`The Fire Mage Analyzer analyzers CombustionFirestarter matches the suggestions snapshot 1`] = `"module inactive"`;
-
-exports[`The Fire Mage Analyzer analyzers CombustionSpellUsage matches the statistic snapshot 1`] = `"module has no statistic method"`;
-
-exports[`The Fire Mage Analyzer analyzers CombustionSpellUsage matches the suggestions snapshot 1`] = `Array []`;
+exports[`The Fire Mage Analyzer analyzers Combustion matches the suggestions snapshot 1`] = `
```

Comment:
Did you change the rules too or was this unintended?

Reply:
I didnt change any of the rules, or anything in the checklist, just the module names

Reply:
So then there's a new bug since the snapshot suggests a new suggestion is being done?

Reply:
i guess so. i didnt even look at what changed in the snapshot, woops